apps/cli/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@webstudio-is/cli",
+  "version": "0.83.0",
+  "description": "Webstudio CLI",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "type": "module",
+  "bin": {
+    "webstudio": "./bin/cli.js"
+  },
+  "scripts": {
+    "dev": "build-package --watch",
```

Comment:
No need dev here. We use it only for remix app dependencies.

Reply:
Yeah, but good to have when working on the `cli` to make changes and comes in handy :)